src/app/shared/components/vm-statistics/vm-statistics.component.ts

```diff
@@ -179,10 +179,14 @@ export class VmStatisticsComponent implements OnInit, OnChanges {
     return this.getStatsStringWithNoRestrictions(value, units, precision);
   }
 
-  public getStatsStringFor(resource: keyof ResourcesData, units?: string): Observable<string> {
+  public getStatsStringFor(
+    resource: keyof ResourcesData,
+    units?: string,
```

Comment:
an optional argument should be the last

Reply:
Ideally, this needs to be refactored with param objects. Will do this in a separate task later.